clang/test/Preprocessor/predefined-macros.c

```diff
@@ -216,14 +216,21 @@
 // CHECK-HIP-DEV: #define __HIP__ 1
 
 // RUN: %clang_cc1 %s -E -dM -fsycl-is-device \
-// RUN:   -triple spir64_fpga-unknown-unknown-sycldevice -o - \
+// RUN:   -triple spir64-unknown-unknown -fintelfpga -o - \
```

Comment:
Could you keep the spir64_fpga triple but without -fintelfpga and CHECK-NO-RANGE test?   This will confirm that we are changing the behavior from before.  i.e. keep the old test, but do the CHECK-NO-RANGE check.

The new variant with spir64 and -fintelfpga can be a separate RUN line in addition.

Reply:
Done. Thanks @premanandrao